caplena/api/api_version.py

```diff
@@ -3,8 +3,9 @@
 
 class ApiVersion(Enum):
     DEFAULT = 0
-    VER_2022_01_01 = 1
-    VER_2022_06_20 = 2
+    VER_2022_11_22 = 3
+    VER_2022_12_22 = 4
+    VER_2023_01_07 = 5
```

Comment:
Do we need a PR in labelhippo which adds that version? Or am I missing something?
https://github.com/search?q=repo%3Acaplena%2Flabelhippoapi%20VER_2023_01_07&type=code